Test/DependencyCollector/FunctionalTests/FuncTest/Helpers/TestWebApplication.cs

```diff
@@ -110,7 +110,8 @@ internal void Deploy(bool enableWin32Mode = false)
             }
             catch (Exception ex)
             {
-                Trace.TraceError("Exception occured while attempting to deploy application {0}: {1}.", this.AppName, ex);
+                Trace.TraceError("Exception occured while attempting to deploy application {0}: {1}. Tests wont continue as they are guaranteed to fail.", this.AppName, ex);
```

Comment:
nitpick: "wont" should be either "will not" or "won't" (I prefer "will not" in this sentence).